dts/bindings/sensor/rohm,bh1750.yaml

```diff
@@ -0,0 +1,40 @@
+#####################################
+# Copyright (c) 2022 Sngular
+#
+# Team: IoT
+# Author: Jeronimo Agullo Ocampos
+#####################################
+
+description: BH1750 ambient light sensor sensor
+
+compatible: "rohm,bh1750"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+
+properties:
+  resolution:
+    type: int
+    default: 1
+    description: |
+      Sensor resolution. Default is 1lx.
+      The values are the corresponding hexadecimal values to 
+      write to configuration register
```

Comment:
Please justify the chosen default value in the description. Is it a power on register reset value?

Reply:
```suggestion
      Sensor resolution. 
      The values are the corresponding hexadecimal values to 
      write to configuration register.
      Default H-resolution mode (1lx) since it is the recommended
      mode in the page 5 of datasheet.
```